docs/api-testing/composer/io-components.md

```diff
@@ -73,26 +74,50 @@ Performs a `POST` method request.
      <td><p><small>| REQUIRED | STRING |</small></p></td>
     </tr>
     <tr>
-     <td><strong>Content</strong></td>
+     <td><strong>Body</strong></td>
      <td><p><small>| OPTIONAL | STRING |</small></p></td>
     </tr>
   </tbody>
 </table>
 
 
+#### File
+1. Click **File (Multi-part)**.<br/><img src={useBaseUrl('img/api-testing/FileMulti.png')} alt="Post" />
+1. Fill in the **Name** and **Value** fields.<br/><img src={useBaseUrl('img/api-testing/FileMulti1.png')} alt="Post" />
+
+<table id="table-api">
+  <tbody>
+  <tr>
+  <td colSpan='2'>Fields</td>
+  </tr>
+    <tr>
+     <td><strong>Name</strong></td>
+     <td><p><small>| REQUIRED | STRING |</small></p></td>
+    </tr>
+    <tr>
+     <td><strong>File</strong></td>
+     <td><p><small>| REQUIRED | STRING |</small></p></td>
+    </tr>
+    <tr>
+     <td><strong>Filename</strong></td>
+     <td><p><small>| REQUIRED | STRING |</small></p></td>
+    </tr>
+  </tbody>
+</table>
+
 ### PUT
-See [Parameters](#param) and [Body](#body).
+See [Param](#param), [Body](#body), and [File](#file).
 
 ### PATCH
-See [Parameters](#param) and [Body](#body).
+See [Param](#param), [Body](#body), and [File](#file).
 
 ### DELETE
-See [Parameters](#param) and [Body](#body).
+See [Param](#param), [Body](#body), and [File](#file).
 
```

Comment:
@aliciakrebs I know this was the same in the old documentation but all POST, PUT DELETE, PATCH can also have "Request Header" (defined later in the page), "Query Params" (defined later in the page), "Config" (defined later in the page) and  "Basic Authentication" (defined later in the page). Can you please add the links also to these sections?

Reply:
Yes, I will do that in a separate PR.